File: oauth2-web-app/src/main/java/com/aeon/controllers/HomeController.java
package com.aeon.controllers;

import org.springframework.http.MediaType;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping(
        produces = MediaType.TEXT_PLAIN_VALUE
)
public class HomeController {

    @GetMapping("/ping")
    public String ping() {
        return "pong";
    }
}

File: spring-boot-gql/src/main/kotlin/com/aeon/gql/controller/HomeController.kt
package com.aeon.gql.controller

import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.web.bind.annotation.GetMapping

@Controller
class HomeController {

    @GetMapping("/")
    fun index(model: Model): String {
        model.addAttribute("message", "Hello There how are you")
        return "index"
    }
}